ballerina/tests/test.bal

```diff
@@ -59,9 +59,9 @@ ConnectionConfig sfConfigCredentialsFlow = {
     }
 };
 
-Client baseClient = check new (sfConfigRefreshFlow);
-Client baseClientPasswordFlow = check new (sfConfigPasswordFlow);
-Client baseClientCredentialsFlow = check new (sfConfigCredentialsFlow);
+Client baseClient = check new (sfRefreshFlowConfig);
```

Comment:
```suggestion
Client refreshFlowClient = check new (sfRefreshFlowConfig);
```

Reply:
Updatd in [fc34a20](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/308/commits/fc34a20589a52dafcfe8c8f47d1f35a7c8d80e2a)